target/generated-sources/annotations/com/example/springresttask/domain/mapper/EmailMapperImpl.java

```diff
@@ -0,0 +1,59 @@
+package com.example.springresttask.domain.mapper;
```

Comment:
.target classes are generated by compiler and must not be pushed into the source repository. Add the entire directory to .gitignore